content/posts/die-digitale-transformation-ist-eine-reise.md

```diff
@@ -0,0 +1,60 @@
+Nach einer ersten Welle von Anwendungsfällen wiederholt sich der Prozess mit der nächsten Welle – und so weiter. Was in den verschiedenen Phasen umgesetzt wird ist nicht nur eine Frage des gesunden Menschenverstandes, sondern auch entscheidend, um die Zustimmung Ihres Teams zu erhalten und die Kosten überschaubar zu halten. Wenn Sie neue Technologien ausprobieren ohne zunächst zu verstehen, warum sie für Ihr spezielles Geschäftsproblem oder Ihre Geschäftseinheit geeignet sind, könnten Sie viel mehr als nötig in Testversionen investieren, die nie von jemandem im Team verwendet werden (und daher nutzlos sind). Andererseits sollten Sie, wenn Sie bahnbrechende Innovationen wie Chatbots oder künstliche Intelligenz (KI) testen, nicht einfach das für bare Münze nehmen, was deren Anbieter Ihnen sagen. 
+
+## Arbeiten Sie Agile
+
+Klassische Projektkonzepte und Projektplanungen haben bei IT-Projekten oft zu Kostenüberschreitungen und Lieferverzögerungen geführt. Dies passiert in den letzten Jahren seltener: Neue Arbeitsmethoden erlauben besser während der Entwicklung auf unbekannte oder unbewusste Komplexitäten einzugehen. Diese Methoden werden unter den Überbegriff “Agile Entwicklung” zusammengefasst. Nicht nur IT-Projekte, sondern das ganze Betriebsmodell einer Firma kann von einer agilen Vorgehensweise profitieren. Damit können Projekte besser auf die individuellen Bedürfnisse der Firmen eingehen.
+Transformieren Sie Arbeitsweisen und das Betriebsmodell. Agile Arbeitsweisen ermöglichen es Unternehmen, schnell auf Veränderungen zu reagieren. Daher ist die Unterstützung von Unternehmen bei der Einführung von Agilität eine wichtige Säule unserer Beratung zur digitalen Transformation. Wir verankern die Verhaltensweisen und die Kultur, die die funktionsübergreifende Zusammenarbeit, iterative Entwicklung und einen neuen Lernansatz fördern – einen, der es Menschen ermöglicht, sich mit digitaler Geschwindigkeit anzupassen und innovativ zu sein. Wir zeigen Führungskräften, wie sie Agilität im gesamten Unternehmen einsetzen können. Und ebenso wichtig ist, dass wir daran arbeiten, Governance, Prozesse und Organisationsstruktur an einem stärker plattformgesteuerten – und weniger isolierten – Betriebsmodell auszurichten.
+
+## Vergessen Sie nicht die Menschen und die Kultur
+
+Dies ist der wichtigste Aspekt der Innovation, insbesondere wenn es um die digitale Transformation geht. Menschen sind nicht nur eine Ressource, sondern auch ein Vermögenswert. Der Erfolg eines Unternehmens hängt von seinen Mitarbeitern ab: Sie sind die Hauptakteure bei der Gestaltung der Unternehmenskultur und tragen zum Erfolg oder Misserfolg des Unternehmens bei. Sie können noch so tolle Ideen haben, wenn Sie kein Team haben, mit dem sie umgesetzt werden können, werden sie nicht zum Tragen kommen.
+Die digitale Transformation ist eine Herausforderung, aber wenn man sie Schritt für Schritt angeht, kann man seine Ziele erreichen. Die digitale Transformation ist eine Reise, kein Ziel.
```

Comment:
```suggestion
Die digitale Transformation ist eine Herausforderung, aber wenn man sie Schritt für Schritt angeht, kann man seine Ziele erreichen. Die digitale Transformation selbst ist eine Reise, kein Ziel an sich.
```
I find it confusing to first say you can reach your goals, and then that transformation is not a goal but a path. I think my suggestion makes that clearer.